src/upgrade.py

```diff
@@ -125,12 +131,20 @@ def _count_online_instances(status_dict: dict) -> int:
     @override
     def log_rollback_instructions(self) -> None:
         """Log rollback instructions."""
+        juju_version = JujuVersion.from_environ()
+        if juju_version.major > 2:
+            run_action = "run"
+            wait = ""
+        else:
+            run_action = "run-action"
+            wait = " --wait"
```

Comment:
JFYI, in documentation I am using "run-action --wait" :-)
https://charmhub.io/mysql/docs/e-juju-details